nifi-toolkit/nifi-toolkit-assembly/pom.xml

```diff
@@ -100,4 +100,87 @@ language governing permissions and limitations under the License. -->
             <scope>compile</scope>
         </dependency>
     </dependencies>
+
+
+    <profiles>
+        <profile>
+            <id>docker</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>copy-resources</id>
```

Comment:
On my machine, this copy execution does not preserve file permissions for `docker-entrypoint.sh`, so when I try to run the image, I get: 

> starting container process caused "exec: \"/opt/sh/docker-entrypoint.sh\": permission denied".

Reply:
not a huge deal but that would cause an additional layer in the image.  will play around with this a bit and see if there is something else that can be adjusted, otherwise will default to that

Thanks!

Reply:
@kevdoran It does appear to be a bug.  Looks like I may have had a cached version in target while doing some work.  Fixed that up and pushed out the change.

Reply:
Thanks @apiri! It should be good now. I will verify locally and update my review.